modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -645,13 +647,20 @@ class Iframe extends React.Component {
 		iframeWindow.document.body.classList.add(CSS_CLASS_IFRAME_BODY);
 
 		if (iframeWindow.Liferay.SPA) {
-			this.beforeScreenFlipHandler = iframeWindow.Liferay.on(
-				'beforeScreenFlip',
-				() => {
+			this.spaNavigationHandlers = [];
+
+			this.spaNavigationHandlers.push(
```

Comment:
Rather than creating the array and then pushing onto it, can we just create the array with the two items? 

so `this.myArray = [itemOne, itemTwo];` instead of `this.myArray = []; this.myArray.push(itemOne, itemTwo)`